pages/goals/[slug].tsx

```diff
@@ -80,11 +82,36 @@ const EditGoalsPage = () => {
                 }));
               }}
             />
+
             {editedGoalValues.goalName.length > 60 && (
               <p className="text-xs text-red-600">
                 please keep your goal under 60 characters
               </p>
             )}
+            <div className="flex">
+              <ArrowCircleRightIcon
```

Comment:
- add a hover state
- move icon to the right of the section title
- use the same icon from the main goals page

Reply:
updated PR to fix these issues!